tests/test_valid.rs

```diff
@@ -92,23 +92,12 @@ fn test_table_reordering() {
 [a.b.c.d]
 [a.b.c]
 [[bin]] # bin 3
-"#;
-    let expected = r#"
-[[bin]] # bin 1
-[[bin]] # bin 2
-[[bin]] # bin 3
-[a]
-[a.b.c]
-[a.b.c.e]
-[a.b.c.d]
-[other.table]
 "#;
     let doc = toml.parse::<Document>();
     assert!(doc.is_ok());
     let doc = doc.unwrap();
 
-    assert_eq!(doc.to_string(), expected);
-    assert_eq!(doc.to_string_in_original_order(), toml);
+    assert_eq!(doc.to_string(), toml);
```

Comment:
I think this test doesn't make sense anymore as there is no reordering and also could update the README to remove limitation 3?